dynamic-crawling/yanolja.py

```
@@ -0,0 +1,196 @@
+
+# In[21]:
+
+
+import pandas as pd
+
+# 별점과 리뷰를 결합하여 리스트 생성
+data = list(zip(ratings, reviews))
+
+# DataFrame으로 변환
+df_reviews = pd.DataFrame(data, columns=['Rating', 'Review'])
+df_reviews
```

Ruff (0.13.1)

Found useless expression. Either assign it to a variable or remove it.

(B018)